tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/TestHelpers.cpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/Evolution/DiscontinuousGalerkin/SlopeLimiters/TestHelpers.hpp"
+
+#include "Domain/Direction.hpp"
+#include "Domain/Element.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/Neighbors.hpp"
+
+namespace SlopeLimiterTestHelpers {
```

Comment:
We probably want to have some uniformity with these namespaces. I personally like nesting them in a `TestHelpers` namespace, e.g.
```cpp
namespace TestHelpers {
namespace SlopeLimiter {
```
I know this isn't a uniform thing in the code yet but it's something I've thought about a few times now. I'm also fine with the opposite scoping of the namespaces but prefer `TestHelpers` being the outer one so that at least all the helpers are quarantined to one location. @wthrowe @kidder what is your opinion on this?

Reply:
yeah, I like that

Reply:
Let's do it.

Reply:
Okay, I will make this change.

It looks like bigger changes to this PR are needed to better accommodate the tests of the WENO limiter, so I will update the namespaces while I make the other changes...